### PR TITLE
Countryside stewardship grants

### PR DESCRIPTION
**Pull request by tommyp**
This PR contains the work necessary to enable creation of DEFRA/Natural England Countryside Stewardship Grants from Specialist Publisher. The facets are still undecided so they will be added at a later date. One thing to note is this scheme is owned by DEFRA and Natural England so this will require some refactoring of the permission checker, but this will also happen at a later date. [Ticket](https://trello.com/c/gZoWid5B/523-finder-countryside-stewardship-grants).

### CODE REVIEW

File: app/observers/countryside_stewardship_grant_observers_registry.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    SpecialistPublisherWiring.get(:countryside_stewardship_grant_rummager_deleter)
+  end
+
+  def content_api_withdrawer
+    SpecialistPublisherWiring.get(:specialist_document_content_api_withdrawer)
+  end
+
+  def publication_alert_formatter(document)
+    CountrysideStewardshipGrantPublicationAlertFormatter.new(
+    url_maker: url_maker,
```
Comment (evilstreak) [inline app/observers/countryside_stewardship_grant_observers_registry.rb]: The indentation is off here

File: app/views/countryside_stewardship_grants/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  <div class="actions">
+    <button name="save" class="btn btn-success">Save as draft</button>
+  </div>
+  <% end %>
+</div>
+
+<%= render partial: "specialist_documents/attachments_form", locals: { document: document } %>
+
+<%= render partial: "specialist_documents/js_preview", locals: { document: document, form_namespace: "cma_case" } %>
```
Comment (evilstreak) [inline app/views/countryside_stewardship_grants/_form.html.erb]: Should `form_namespace` be something different to `cma_case`?

File: finders/metadata/countryside-stewardship-grants.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "slug": "countryside-stewardship-grants",
+  "format": "countryside_stewardship_grants",
```
Comment (evilstreak) [inline finders/metadata/countryside-stewardship-grants.json]: I think format needs to be singular

**Comment by evilstreak** [comment]
Looks good :+1: 

I mean, it's a lot of copy-pasting, which suggests we could probably make things better in many places. But this will get the job done.

**Comment by tommyp** [comment]
@evilstreak I've already amended the commits you commented on. This should be good to go now.

**Pull request merged**